pilot/pkg/networking/grpcgen/lds.go

```diff
@@ -298,6 +298,11 @@ func buildOutboundListeners(node *model.Proxy, push *model.PushContext, filter l
 				if features.PersistentSessionLabel != "" {
 					sessionCookie := sv.Attributes.Labels[features.PersistentSessionLabel]
 					if sessionCookie != "" {
+						cookieNamePath := strings.Split(sessionCookie, ":")
```

Comment:
nit: `strings.Cut` is preferred for splitting at a single element, although the behavior should be the same in this case